types/index.d.ts

```diff
@@ -1,5 +1,431 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
+interface CustomMatchers {
+  /**
+   * Note: Currently unimplemented
+   * Passing assertion
+   *
+   * @param {String} message
+   */
+  pass(message: string): void;
```

Comment:
why are these changed from `R` to `void`?

Reply:
Weird, I just copy pasted

Reply:
I'll do it again in a few minutes

Reply:
Never mind, there shouldn't be any `R` I copy pasted from the Expect interface